rosplan_planning_system/launch/includes/problem_interface.launch

```diff
@@ -15,6 +16,7 @@
 		<param name="domain_path"    value="$(arg domain_path)" />
 		<param name="problem_path"   value="$(arg problem_path)" />
 		<param name="problem_topic"  value="$(arg problem_topic)" />
+		<param name="planning_language"  value="$(arg planning_language)" />
```

Comment:
planning_language vs. planner_language (as it is declared above)?

Reply:
Fixed it this morning